api/v1alpha1/common_types.go

```diff
@@ -591,6 +602,53 @@ type VaultSecretReference struct {
 	// The Vault key of the secret
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	Key string `json:"key"`
+	// The Vault secret store reference
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +optional
+	SecretStoreRef *VaultSecretStoreReferenceSpec `json:"secretStoreRef,omitempty"`
+	// The Vault refresh interval of the secret (seconds)
```

Comment:
```suggestion
	// The Vault refresh interval of the secret (duration)
```

Applies on all instances of this description

Reply:
Fixed https://github.com/3scale-ops/saas-operator/pull/150/commits/5315aa660a6c15a8713d9f9d510e5bbcdba14460